int64 a_operation_takeoff_height
---
int64 a_operation_status
